app/views/categories/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="h-screen px-5">
+<h1 class="text-center">Welcome, <%= current_user.name %>!</h1>
+<!-- List categories here -->
+<% @categories.each do |category| %>
+  <div class="flex flex-col">
+    <!-- card -->
+    <div v-for="card in cards" class="flex flex-row overflow-hidden
+                                        bg-white rounded-lg shadow-xl mt-4 w-100 mx-2">
+      <!-- media -->
+      <div class="h-full w-auto">
+        <%= image_tag category.icon, alt: 'Category Icon', class: "inset-0 w-full object-cover object-center" if category.icon.present?%>
+      </div>
+      <!-- content -->
+      <div class="flex flex-col justify-evenly w-full py-4 px-6 text-gray-800">
+        <div class="flex justify-between mt-8">
+          <p class="font-semibold text-lg leading-tight truncate"><%= category.name %></p>
+          <p class="font-semibold text-lg">$ <%= category.total_amount %></p>
+        </div>
+        <p class="text-sm text-gray-700 uppercase tracking-wide font-semibold mt-2">
+          <%= time_ago_in_words(category.created_at) %>
+        </p>
+      </div>
+      <p>
+    </div>
+  </div>
+    <% if can? :read, @category %>
+    <%= link_to "View Transactions", category_transactions_path(category), class: 'actions group relative w-25 flex
+         justify-center py-2 px-4 border border-transparent mt-4 text-sm font-medium
+         rounded-md text-white bg-indigo-600 hover:bg-indigo-700 cursor:pointer focus:outline-none
+         focus:ring-2 focus-2 focus-offset-2 focus:ring-indigo-500'%>
+    <% end %>
+  <% end %>
+  <%= link_to "Add a New Category", new_category_path,class: 'actions group relative flex
+         justify-center py-2 px-4 border border-transparent mt-4 text-sm font-medium
+         rounded-md text-white bg-indigo-600 hover:bg-indigo-700 cursor:pointer focus:outline-none
+         focus:ring-2 focus-2 focus-offset-2 focus:ring-indigo-500' %>
+</div>
```

Comment:
- [ ] Great job for creating the categories page but I think the UX is not the same as shown in the given design since the color of the button is green and we should stick with the given colors as mentioned in the requirements 💯✅
![image](https://github.com/lincoln1883/piggy-bank/assets/58553711/d6669de9-a5ed-4dd3-b67e-bdceccedc68c)

- [ ] I think it would be great if you just kept the given layout without adding the button for view transactions to each category but instead, you can link the category itself to the transaction page since we can't see a button included after each category so kindly just link it as I said 💯🙏🏻✨✅

![image](https://github.com/lincoln1883/piggy-bank/assets/58553711/d8349c48-96cb-41cc-b15e-8be96a7e4ea3)




### Here the color is different so kindly stick with the given colors ♻


- Your
![image](https://github.com/lincoln1883/piggy-bank/assets/58553711/a440abde-5bc4-4181-9947-361287519966)

![image](https://github.com/lincoln1883/piggy-bank/assets/58553711/d0264c39-67ae-4785-8c33-5b3551f22c5a)

- Given
![image](https://github.com/lincoln1883/piggy-bank/assets/58553711/a393e75d-103c-4ff1-8b6c-0a8fc193fcdc)